WalletWasabi/Models/ChaumianCoinJoin/CcjRound.cs

```diff
@@ -771,7 +771,7 @@ private async Task TryOptimizeConfirmationTargetAsync(ISet<uint256> transactionH
 				// If the transaction does not spend unconfirmed coins then the confirmation target can be the one that's been set in the config.
 				var originalConfirmationTarget = AdjustedConfirmationTarget;
 
-				// Note that only dependents matter, spenders does not matter much or at all, they just make this transaction to be faster to confirm faster.
+				// Note that only dependents matter, spenders does not matter much or at all, they just make this transaction faster to confirm faster.
```

Comment:
> hey just make this transaction faster to confirm faster.

Is the first "faster" not redundant?  

I probably misunderstand the context here as I don't understand the code, but perhaps the part of the sentence should be: `they just allow this transaction to be confirmed faster` or similar?